src/middleware/navigation/has.overseas.entity.middleware.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response, NextFunction } from 'express';
+
+import { logger } from '../../utils/logger';
+import { UPDATE_LANDING_URL } from '../../config';
+import { getApplicationData } from "../../utils/application.data";
+import { checkOverseasEntityDetailsEntered, NavigationErrorMessage } from './check.condition';
+
+export const hasOverseasEntity = (req: Request, res: Response, next: NextFunction): void => {
```

Comment:
hasOverseasEntityNumber - checks for number but not having found it and could be confused with entity details entered